### PR TITLE
 Support for AWS elasticsearch

### PR DESCRIPTION
**Pull request by pgvishnuram**
* integration with aws elasticsearch
* support for aws keys
* general configs

## 📋 Checklist

- [ ] The PR title is informative to the user experience

### CODE REVIEW

**Comment by pgvishnuram** [comment]
@rbankston - i have tested with AWS CREDS and AWS IAM as well - can you have a look once

File: charts/external-es-proxy/values.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ images:
     repository: openresty/openresty
     pullPolicy: IfNotPresent
     tag: latest
+  awsproxy:
+    repository: abutaha/aws-es-proxy
```
Comment (rbankston) [inline charts/external-es-proxy/values.yaml:13]: Saw the license which is permissive. Should we package and use an astronomer/ap-aws-es-proxy  so in the field we don't hit docker hub issues and also are able to verify the provenance of the image? @danielhoherd

**Review by rbankston — APPROVED** [review]
The code looks good to me. I just had a comment about the docker image from aws-es-proxy to include it as an ap type image.

File: charts/external-es-proxy/values.yaml (hunk 1 above)
Comment (danielhoherd) [reply charts/external-es-proxy/values.yaml:13]: yes, absolutely. we should vendor this into ap-vendor.

Comment (pgvishnuram) [reply charts/external-es-proxy/values.yaml:13]: will be adding it to ap-vendor

**Pull request merged**